### DISCUSSION TITLE
Dealing with font faces

### DISCUSSION THREAD

**Discussion by Code-Victor** [stitchesjs/stitches#1019] [Help] [answered]
How can I use google fonts in my `@font-face` declaration? 
I also tried using a font file(.tff) and it didn't work
```javascript
const globalStyles = globalCss({
  '@font-face':{
    fontFamily:'Open Sans',
    src:`url('./fonts/OpenSans-VariableFont.ttf') format('truetype')`,

  },
})
```
declaration 
```javascript
const Button = styled("button", {
  backgroundColor: "$gray400",
  borderRadius: "9999px",
  fontSize: "16px",
  fontFamily: "Open Sans",
})
```
how  can I do this ?

**Comment by tianenpang** [comment, marked as answer]
I am using different fonts as [theme token](https://stitches.dev/docs/tokens) and then using them in `styled` or `css`.

- Specify the font in [`globalCss`](https://stitches.dev/docs/styling#global-styles).
- Define the font token in [`createStitches`](https://stitches.dev/docs/tokens#defining-tokens).

Here's an example with next.js: [codesandbox.io/s/next-stitches-font-1ygp26](https://codesandbox.io/s/next-stitches-font-1ygp26?file=/styles/stitches.config.ts)

[Preload](https://leerob.io/blog/fonts#preloading) the font in [`pages/_document.tsx`](https://nextjs.org/docs/advanced-features/custom-document)

**Reply by Code-Victor** [reply]
Thank you..... It works 💯
